src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -2064,9 +2131,41 @@ NEVER_INLINE Value ByteCodeInterpreter::callFunctionInWithScope(ExecutionState&
         receiverObj = bindedRecord->asObjectEnvironmentRecord()->bindingObject();
     else
         receiverObj = state.context()->globalObject();
+
+    if (code->m_hasSpreadElement) {
+        size_t argc = 0;
+        Value* newArgv = spreadFunctionArguments(state, argv, code->m_argumentCount, &argc);
+        return FunctionObject::call(state, callee, receiverObj, argc, newArgv);
+    }
     return FunctionObject::call(state, callee, receiverObj, code->m_argumentCount, argv);
 }
 
+Value* ByteCodeInterpreter::spreadFunctionArguments(ExecutionState& state, const Value* argv, const size_t argc, size_t* newArgc)
+{
+    std::vector<Value, GCUtil::gc_malloc_ignore_off_page_allocator<Value>>* argVector = new std::vector<Value, GCUtil::gc_malloc_ignore_off_page_allocator<Value>>();
```

Comment:
How about locally allocating an `ValueVector` instead of dynamically allocating the vector? Then, call site (caller) can copy the value of returned `ValueVector` in its local vector and use the vector for argv. Otherwise, when the vector is dynamically allocated, its values will not be reclaimed until the GC called.

Reply:
It's fine for me.

Reply:
@rerobika Thanks!